app/assets/stylesheets/custom.scss

```diff
@@ -2,3 +2,12 @@
 //
 // You can also create SCSS files in the app/assets/stylesheets/custom/
 // folder and they will be loaded automatically.
+
+.top-bar {
+  .top-bar-title a img {
+    @include breakpoint(medium) {
+      width: 7rem;
+      margin: 0;
```

Comment:
Isn't the margin already zero without this line? :thinking: Defined in `app/assets/stylesheets/layout.scss`, line 666.